build-aux/modules/90-carla.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "carla",
+    "buildsystem": "cmake-ninja",
+    "builddir": true,
+    "subdir": "cmake",
+    "config-opts": [
+        "-DCARLA_SKIP_LIBDIR_SUFFIX:BOOL=ON",
+        "-DCARLA_USE_JACK:BOOL=OFF",
+        "-DCARLA_USE_OSC:BOOL=OFF"
+    ],
+    "sources": [
+        {
+            "type": "git",
+            "url": "https://github.com/falkTX/Carla.git",
+            "commit": "8d4bf13cd411307649f3e4327d855d98f06fb76f"
```

Comment:
This should be aligned with obs-deps, or a good reason of why it is not must be provided

Reply:
the main reason is obvious, updating obs-deps requires going through another PR, and we still do not have a release on obs-deps side to even begin to verify this side.
so pretty much